nautilus_trader/adapters/dydx/execution.py

```diff
@@ -327,6 +340,14 @@ async def _connect(self) -> None:
 
         await self._set_leverage()
 
+        if self._reconciliation_interval_secs:
```

Comment:
This _may_ not be necessary now because of this new continuous position reconciliation on `develop` branch:
- https://github.com/nautechsystems/nautilus_trader/blob/develop/nautilus_trader/live/config.py#L145